plugins/cody-chat/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,7 @@ Require-Bundle: org.eclipse.core.runtime,org.eclipse.jdt.annotation,org.
  ipse.jetty.io,org.eclipse.jetty.server,org.eclipse.jetty.util,org.eclip
  se.lsp4j,org.eclipse.ui,org.eclipse.ui.ide.application,com.google.gson,
  org.eclipse.lsp4j.jsonrpc,org.eclipse.ui.ide,org.eclipse.core.resources
- ,org.eclipse.ui.editors,org.eclipse.jface.text
+ ,org.eclipse.ui.editors,org.eclipse.jface.text,org.eclipse.core.filesystem

```

Comment:
@olafurpg I just added this manually so I'm unsure if this will break the next release.